File: 프로그래머스/1/77484. 로또의 최고 순위와 최저 순위/로또의 최고 순위와 최저 순위.py
def solution(lottos, win_nums):
    rank = [6,6,5,4,3,2,1]
    curN = len([v for v in lottos if v in win_nums])
    zeroN = lottos.count(0)

    return [rank[curN+zeroN], rank[curN]]

File: 프로그래머스/1/77484. 로또의 최고 순위와 최저 순위/로또의 최고 순위와 최저 순위.js
function solution(lottos, win_nums) {
    const rank = [6,6,5,4,3,2,1]
    let curN = 0;
    let zeroN = 0
    
    for (let v of lottos){
        if (v === 0) zeroN++
        else if (win_nums.includes(v)) curN++
    }
    
    return [rank[curN+zeroN],rank[curN]];
}